opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -1192,6 +1192,20 @@ namespace Opm
         }
     }
 
+    template <typename TypeTag>
+    void
+    WellInterface<TypeTag>::
+    setPrevSurfaceRates(WellState& well_state, 
```

Comment:
This function can go to WellInterfaceGeneric.

Reply:
Done.